docs/policies.md

```diff
@@ -130,20 +130,20 @@ Read more about hostPath in the [official Kubernetes documentation](https://kube
 
 This policy checks for the existence of a Privileged Pod Security Context within the workload configuration and will deny the workload if detected.
 
-Pods can enable privileged mode, using the privileged flag on the SecurityContext of the container spec. This configuration allows for many unintended security related vulnerabilities and should therefore be disallowed on clusters. This is a pretty basic check, and there are certainly more ways to abuse a pods security context, but this should provide a base to start from to build a more advanced policy.
+Pods can enable privileged mode, using the privileged flag on the SecurityContext of the container spec. This configuration allows for many unintended security-related vulnerabilities and should therefore be disallowed on clusters. This is a pretty basic check, and there are certainly more ways to abuse a pods security context, but this should provide a base to start from to build a more advanced policy.
 
 Read more about Privileged Pods in the [official Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/pods/pod/#privileged-mode-for-pod-containers)
 
 ## Node Port Range (Check ID: MT2002)
 
 This policy enforces that a nodePort configured in a Service falls within the nodePort range that is defined in the corresponding namespace annotation.
 
-NodePorts are used as a way to expose a Service external to a cluster. Since NodePorts are a finite resource, this policy aims to control NodePort usage via an allow list model using an annotation on a Namespace to designate the allowable NodePort values/range.
+NodePorts are used as a way to expose a Service external to a cluster. Since NodePorts are a finite resource, this policy aims to control NodePort usage via an allow list model using annotation on a Namespace to designate the allowable NodePort values/range.
 
 The configuration should follow these recommended rules:
 
 The nodePort range annotation can be a single number, numbers separated by commas, a range split with a hyphen, or a combination of the three
 The nodePort annotation on the namespace should be "k8s.t-mobile.com/nodeportRange"
-Set the annotation to "na" if no nodePort range will to be set, that is seen as an exception value
+Set the annotation to "na" if no nodePort range will to be set, that is seen as an exceptional value
```

Comment:
issue(blocking): `exception` in the original text is accurate within this context, but there are a couple of other items that could be addressed here:

```suggestion
Set the annotation to "na" if no nodePort range will not be set, that is seen as an exception value
```

Reply:
chore (non-blocking): While it wasn't specifically called out in the original issue, there's a formatting issue on this section. 

The lines following the `The configuration should follow these recommended rules:` line should either have a newline in between each of them, or moved to an unordered list. Without this, the markdown is rendered as a continuous sentence with no punctuation.

Reply:
I can't understand what I have to change here. Can you please elaborate?

Reply:
The highlighted changes in the suggestion above at a minimum. 

It's non-blocking, but would be appreciated to add a newline between each line following the `The configuration should follow these recommended rules:` line.

Right now it's like this:

```
The nodePort range annotation can be a single number, numbers separated by commas, a range split with a hyphen, or a combination of the three
The nodePort annotation on the namespace should be "k8s.t-mobile.com/nodeportRange"
Set the annotation to "na" if no nodePort range will to be set, that is seen as an exception value
Set the annotation to "na" if no nodePort range will not be set, that is seen as an exception value
```

And should be:

```
The nodePort range annotation can be a single number, numbers separated by commas, a range split with a hyphen, or a combination of the three

The nodePort annotation on the namespace should be "k8s.t-mobile.com/nodeportRange"

Set the annotation to "na" if no nodePort range will to be set, that is seen as an exception value

Set the annotation to "na" if no nodePort range will not be set, that is seen as an exception value
```